orion/pkg/domains/user.go

```diff
@@ -0,0 +1,71 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/sql_helper"
+	"regexp"
+	"time"
+)
+
+var TABLE_USERS = "users"
+
+type User struct {
+	Id         uint
+	CreatedAt  time.Time           `db:"created_at"`
+	UpdatedAt  time.Time           `db:"updated_at"`
+	DeletedAt  sql.NullTime        `db:"deleted_at"`
+	FirstName  string              `json:"firstName"`
+	LastName   string              `json:"lastName"`
+	MiddleName sql.NullString      `json:"middleName"`
+	Email      string              `json:"email"`
+	Phone      string              `json:"phone"`
+	IsGuardian bool                `json:"isGuardian"`
+	GuardianId sql_helper.NullUint `db:"guardian_id" json:"guardianId"`
```

Comment:
For now turn this into a uint. And do the same as above. NullUint only in the Insert() and Update() layer of repo please.
Thank you!